capstone/src/main/resources/static/script.js

```diff
@@ -172,7 +182,7 @@ function getEmail() {
   
   // Temp Back up
   if (email == "") {
-      email = "jenny@google.com";
+      email = "jennysheng@google.com";
   }
```

Comment:
Feel free to change the email to whatever you want, but I would not use a "real email" because if we are testing on the dev server, it could override the data on the actual server where we have to use a real email.

Reply:
oh yeah I just needed an email that exists in an individual's entity or else it it causes some errors. I can make a dummy individual for local use